### PR TITLE
Added Get Started Guide for Raspbian

### PR DESCRIPTION
**Pull request by avladimi**
Added the Get Started Guide for Raspbian

### CODE REVIEW

File: docs/doxygen/openvino_docs.xml [hunk 1]
```diff
@@ -38,6 +38,7 @@
                 <tab type="user" title="Linux" url="@ref openvino_docs_get_started_get_started_linux"/>
                 <tab type="user" title="Windows" url="@ref openvino_docs_get_started_get_started_windows"/>
                 <tab type="user" title="macOS" url="@ref openvino_docs_get_started_get_started_macos"/>
+				<tab type="user" title="Raspbian" url="@ref openvino_docs_get_started_get_started_raspbian"/>
```
Comment (dkurt) [inline docs/doxygen/openvino_docs.xml:41]: Please fix indentation

File: docs/get_started/get_started_raspbian.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+   ```sh
+   git clone --depth 1 https://github.com/openvinotoolkit/open_model_zoo
+   cd open_model_zoo/tools/downloader
+   python3 -m pip install -r requirements.in
+   python3 downloader.py --name face-detection-adas-0001 
+   ```
+
+4. Run the sample, specifying the model and path to the input image:
+```sh
+./armv7l/Release/object_detection_sample_ssd -m face-detection-adas-0001.xml -d MYRIAD -i <path_to_image>
```
Comment (dkurt) [inline docs/get_started/get_started_raspbian.md:83]: Same steps are described at https://github.com/openvinotoolkit/openvino/blob/master/docs/install_guides/installing-openvino-raspbian.md#build-and-run-object-detection-sample

File: docs/get_started/get_started_raspbian.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+<details>
+    <summary><strong>Click for the <code>deployment_tools</code> directory structure</strong></summary>
+   
+
+| Directory&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                                           |  
+|:----------------------------------------|:--------------------------------------------------------------------------------------|
+| `inference_engine/`                     | Inference Engine directory. Contains Inference Engine API binaries and source files, samples and extensions source files, and resources like hardware drivers.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`external/`     | Third-party dependencies and drivers.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`include/`      | Inference Engine header files. For API documentation, see the [Inference Engine API Reference](./annotated.html). |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`lib/`          | Inference Engine static libraries.|
```
Comment (dkurt) [inline docs/get_started/get_started_raspbian.md:35]: Why static?

File: docs/get_started/get_started_raspbian.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+2. Build the Object Detection Sample with the following command:
+```sh
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino/deployment_tools/inference_engine/samples/cpp
+```
+```sh
+make -j2 object_detection_sample_ssd
+```
+3. Download the pre-trained Face Detection model with the Model Downloader:
+
+   - To download the `.bin` file with weights:
```
Comment (dkurt) [inline docs/get_started/get_started_raspbian.md:73]: It also downloads `.xml`

File: docs/get_started/get_started_raspbian.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+|:----------------------------------------|:--------------------------------------------------------------------------------------|
+| `inference_engine/`                     | Inference Engine directory. Contains Inference Engine API binaries and source files, samples and extensions source files, and resources like hardware drivers.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`external/`     | Third-party dependencies and drivers.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`include/`      | Inference Engine header files. For API documentation, see the [Inference Engine API Reference](./annotated.html). |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`lib/`          | Inference Engine static libraries.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`samples/`      | Inference Engine samples. Contains source code for C++ and Python* samples and build scripts. See the [Inference Engine Samples Overview](../IE_DG/Samples_Overview.md). |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`share/`        | CMake configuration files for linking with Inference Engine.|
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`src/`          | Source files for CPU extensions.|
```
Comment (ilya-lavrenov) [inline docs/get_started/get_started_raspbian.md:38]: cpu extensions are obsolete. I suppose we should remove this line

File: docs/get_started/get_started_raspbian.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+```
+The application outputs an image (`out_0.bmp`) with detected faced enclosed in rectangles.
+
+## <a name="basic-guidelines-sample-application"></a>Basic Guidelines for Using Code Samples
+
+Following are some basic guidelines for executing the OpenVINO™ workflow using the code samples:
+
+1. Before using the OpenVINO™ samples, always set up the environment: 
+```sh
+source /opt/intel/openvino/bin/setupvars.sh
```
Comment (dkurt) [inline docs/get_started/get_started_raspbian.md:93]: `<INSTALL_DIR>/bin/setupvars.sh`

File: docs/get_started/get_started_raspbian.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+``` 
+2. Have the directory path for the following:
+- Code Sample binaries located in `~/inference_engine_cpp_samples_build/intel64/Release`
```
Comment (dkurt) [inline docs/get_started/get_started_raspbian.md:96]: Not sure that we have this folder

File: docs/get_started/get_started_raspbian.md (hunk 2 above)
Comment (avladimi) [reply docs/get_started/get_started_raspbian.md:81]: @dkurt Installation Guide will be updated by 2021.2: these instructions will be removed, link to GSG will be added

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**